<launch>
  <node name="robotx_twist_controller_node" pkg="robotx_twist_controller" type="robotx_twist_controller" output="screen">
    <param name="tread" value="2.07" type="double"/>
    <param name="max_left_vel" value="1.0" type="double"/>
    <param name="max_right_vel" value="1.0" type="double"/>
    <param name="kp_vel" value="2.0" type="double"/>
    <param name="ki_vel" value="0.5" type="double"/>
    <param name="kp_ang_vel" value="1.0" type="double"/>
    <param name="ki_ang_vel" value="3.0" type="double"/>
    <param name="publish_rate" value="50" type="double"/>
    <remap from="/carrot_planner_node/twist_cmd" to="/twist_vel"/>
  </node>
</launch>
